dts/arm/nxp/nxp_ke1xz.dtsi

```diff
@@ -24,6 +24,32 @@
 			compatible = "arm,cortex-m0+";
 			clock-frequency = <48000000>;
 			reg = <0>;
+			cpu-power-states = <&idle &stop &pstop1 &pstop2>;
+		};
+
+		power-states {
+			idle: idle {
+				compatible = "zephyr,power-state";
+				power-state-name = "runtime-idle";
+			};
+
```

Comment:
Do we not have any better state name then "suspend-to-idle" for each of the next three?

Reply:
Align with ke1xf,rtxx, and rw6xx platforms, these states names are "suspend-to-idle", and looking at the power binding file, there seems to be no better name.